api/queues.js

```diff
@@ -49,19 +49,23 @@ router.post(
     ]),
     failIfErrors,
   ],
-  (req, res, _next) => {
+  async (req, res, _next) => {
     const { id: courseId } = res.locals.course
     const data = matchedData(req)
 
-    const queue = Queue.build({
+    const queue = await Queue.create({
       name: data.name,
       location: data.location,
       fixedLocation: data.fixedLocation === true,
       courseId,
       createdByUserId: res.locals.userAuthn.id,
     })
 
-    queue.save().then(newQueue => res.status(201).json(newQueue))
+    Queue.scope('questionCount')
+      .findOne({
```

Comment:
You can use `findById(1234)` to make this a little more concise.

Reply:
Done, please check